src/components/UserList/index.tsx

```diff
@@ -110,6 +110,29 @@ const UserList: React.FC = () => {
   const [selectedUsers, setSelectedUsers] = useState<number[]>([]);
   const { user: currentUser } = useUser();
 
+  useEffect(() => {
+    const displayString = window.localStorage.getItem(
+      'userlist-display-settings'
+    );
+
+    if (displayString) {
+      const displaySettings = JSON.parse(displayString);
+
+      setCurrentSort(displaySettings.currentSort);
+      setCurrentPageSize(displaySettings.currentPageSize);
+    }
+  }, []);
+
+  useEffect(() => {
+    window.localStorage.setItem(
+      'userlist-display-settings',
+      JSON.stringify({
+        currentSort,
+        currentPageSize,
+      })
+    );
+  }, [currentSort, currentPageSize]);
```

Comment:
```suggestion
  useEffect(() => {
    const filterString = window.localStorage.getItem('ul-filter-settings');

    if (filterString) {
      const filterSettings = JSON.parse(displayString);

      setCurrentSort(filterSettings.currentSort);
      setCurrentPageSize(filterSettings.currentPageSize);
    }
  }, []);

  useEffect(() => {
    window.localStorage.setItem(
      'ul-filter-settings',
      JSON.stringify({
        currentSort,
        currentPageSize,
      })
    );
  }, [currentSort, currentPageSize]);
```
We should probably follow the same naming pattern used for the request list.

Reply:
I don't think this should be `x-filter-settings` since this does not only save the filter/sort, but the page size too.
I can rename it to `ul-display-settings` though.

Reply:
@danshilm Page size is actually also a filter, however.  Same as with the requests list?